pkg/cache/cache_test.go

```diff
@@ -198,6 +197,19 @@ var _ = Describe("Informer Cache", func() {
 				Expect(actual.Namespace).To(Equal(testNamespaceOne))
 			})
 
+			It("should be able to list objects with GVK populated", func() {
+				By("listing pods")
+				listObj := &kcorev1.PodList{}
+				Expect(informerCache.List(context.Background(), listObj)).To(Succeed())
+
+				By("verifying that the returned pods have GVK populated")
+				Expect(listObj.Items).NotTo(BeEmpty())
+				Expect(listObj.Items).Should(HaveLen(3))
+				for _, p := range listObj.Items {
+					Expect(p.GroupVersionKind().Empty()).To(BeFalse())
```

Comment:
this should test that we list the right gvk (the `corev1` package should have the package group-version as a package-level variable)

Reply:
Done.